pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -2427,7 +2429,6 @@ class Build {
                                     }
                                 }
                             } else {
-                                dockerImageDigest = dockerImageDigest.replaceAll("\\[", "").replaceAll("\\]", "")
```

Comment:
> Can you explain a bit about the "code improvement"? Is this something we should push up to Adopt?

@AdamBrousseau This line does a workaround to remove brackets. But in this PR with the code change below, we do not need to do that as the value retuerns clear:

```
dockerImageDigest = context.sh(script: "docker inspect --format='{{index .RepoDigests 0}}
```